community/kernel/src/main/java/org/neo4j/kernel/impl/api/StateHandlingStatementOperations.java

```diff
@@ -120,6 +119,22 @@ public StateHandlingStatementOperations(
     }
 
     // <Cursors>
+
+    @Override
+    public Cursor<NodeItem> loadNodeById( KernelStatement statement, long nodeId ) throws EntityNotFoundException
```

Comment:
From what i can see all methods in `StateHandlingStatementOperations` have names in form of [entity][operation][additional_criteria] like `nodeCursorGetFromUniqueIndexSeek` or `nodeCursorGetAll`. Would like new methods where following the same principle.